nixpkgs/default.nix

```diff
@@ -1,4 +1,4 @@
 import (fetchTarball {
-   url = https://github.com/NixOS/nixpkgs/archive/a2075a2c314ccb3df6522dd957bb4e237446dc49.tar.gz;
-   sha256 = "1jbnjw06mca7867q81v3i3jljj0wpm38y9zjwij3nd6pp2bqhvbr";
+   url = https://github.com/guibou/nixpkgs/archive/f612ab79002c546d668c4f4ee3d5321fc814b838.tar.gz;
```

Comment:
You mentioned the update is merged, can this point to upstream again?

Reply:
I missed that. Sorry. This also explains the CI failure on nixpkgs.

Done. Waiting for CI.